configure

```diff
@@ -147,6 +147,8 @@ build_config()
 		echo export prefix=$prefix
 		echo export libdir=$libdir
 		echo export bindir=$prefix/bin
+        [ -z ${PKG_CONFIG} ] && PKG_CONFIG="pkg-config"
```

Comment:
I haven't tried this patch myself. But I wonder if the "&&" should be "||" to get the effect you are looking for.